scripts/build.sh

```diff
@@ -21,12 +21,10 @@
 dir=$(dirname $0)
 projectRoot=${dir}/..
 
-if [ -z "$DOCKER_NAMESPACE" ]; then
-  set -- $@ "-Ddocker.project.namespace=$DOCKER_NAMESPACE"
-fi
+source ${projectRoot}/scripts/utils/maven.sh
 
-pushd $projectRoot
-mvn -P-local-docker-build -P-test.local "$@" clean install
+pushd ${projectRoot}
+  maven_utils::execute $@ clean install
```

Comment:
I don't think it's necessary to pass all arbitrary arguments to this script into maven like this - I actually think it adds more potential to confuse people. I'd prefer if this script only passed in the maven arguments/properties that are necessary.